package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "developer-license-donotus",
+  "version": "1.0.0",
+  "description": "Lorem ipsum...",
+  "main": "index.js",
+  "directories": {
+    "lib": "lib",
+    "test": "test"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/smatthewenglish/developer-license-donotus.git"
```

Comment:
need to update url here to be DIMO

Reply:
this needs to be the github url for the project afaik